src/Plugin.cpp

```diff
@@ -141,7 +150,7 @@ void Plugin::init() {
           tool->setSleekness(mPluginSettings.mPolygon.mSleekness);
           mTools.push_back(tool);
         } else if (mNextTool != "none") {
-          std::cout << mNextTool << " is not implemented yet." << std::endl;
+          spdlog::error("Failed to create tool '{}': This is an unknown tool type!", mNextTool);
```

Comment:
Should this be a warning?
```suggestion
          spdlog::warn("Failed to create tool '{}': This is an unknown tool type!", mNextTool);
```